iron-fit-behavior.js

```diff
@@ -12,6 +12,46 @@ import '@polymer/polymer/polymer-legacy.js';
 
 import {dom} from '@polymer/polymer/lib/legacy/polymer.dom.js';
 
+// IE11 has a bug where an element with (1) `overflow: auto;`, (2) size based on
+// its content's natural size, (3) absolute positioning (either `absolute` or
+// `fixed`), and (4) use `max-width` to constrain its width will place its
+// vertical scrollbar outside the area constrained by `max-width`.
+//
+// Unlike other browsers, IE11 doesn't support changing the size of scrollbars
+// with CSS, so we don't need to read this value live from the element in
+// question when trying to work around the bug.
+const getVerticalScrollbarMaxWidthBugOffset = (() => {
```

Comment:
Why the IIFE? Since this is a module, `offset` can be a top-level sibling declaration to `getVerticalScrollbarMaxWidthBugOffset`

Reply:
Yeah, not strictly necessary, I removed it.